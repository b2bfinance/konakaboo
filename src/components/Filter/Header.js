import styled from 'styled-components';

export default styled.div`
  align-items: center;
  background-color: ${props => props.theme.filterHeaderBackground};
  color: ${props => props.theme.filterHeaderColor};
  display: flex;
  font-weight: ${props => props.theme.mainBoldFontWeight};
  justify-content: space-between;
  min-width: 15rem;
  padding: 0.45rem 0.5rem 0.45rem 0.75rem;

  svg {
    fill: ${props => props.theme.filterHeaderColor};
  }
`;
