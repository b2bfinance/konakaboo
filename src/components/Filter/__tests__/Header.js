import React from 'react';
import Header from '../Header';
import { render } from 'enzyme';

describe('Header', () => {
  test('renders correctly', () => {
    expect(render(<Header />)).toMatchSnapshot();
  });
});
