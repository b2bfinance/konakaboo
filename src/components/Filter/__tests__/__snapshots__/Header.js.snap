// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header renders correctly 1`] = `
.c0 {
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-box-pack: justify;
  -webkit-justify-content: space-between;
  -ms-flex-pack: justify;
  justify-content: space-between;
  min-width: 15rem;
  padding: 0.45rem 0.5rem 0.45rem 0.75rem;
}

<div
  class="c0"
>
  
</div>
`;
